### PR TITLE
Scale ordering rather than filtering for htmlpopup and feature services

### PR DESCRIPTION
**Pull request by loicgasser**
@gjn @procrastinatio please review. (possibly before the 22nd :) )

[Test Link](https://mf-geoadmin3.dev.bgdi.ch/?api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fgal_scale_html&lang=fr)

A Bod ID and Feature ID is all we should need to retrieve a correct model. 

mapExtent and imageDisplay parameters are provided in geoadmin when requesting:

- a feature
- a htmlpopup content

Note that we alos need to fix (code to generate this list is in this PR):
https://github.com/geoadmin/mf-chsdi3/issues/2456

Right now we filter out models that are not suitable.
With this PR we use the models that are the most suitable first, and stop when we find the first match.

### CODE REVIEW

File: chsdi/models/__init__.py [hunk 1]
```diff
@@ -74,10 +74,20 @@ def oereb_models_from_bodid(bodId, scale=None):
...
     models = bodmap.get(bodId)
     if models:
-        if scale is not None:
+        if orderScale is not None:
+            mdls = []
+            for m in models:
+                if orderScale < max_scale(m) and orderScale  >= min_scale(m):
+                    mdls.insert(0, m)
+                else:
+                    mdls.append(m)
```
Comment (procrastinatio) [inline chsdi/models/__init__.py]: I don't get it, in the PR description, you say 'break on first match', and you insert anyway all models ?

**Comment by procrastinatio** [comment]
What is _models_scan.py_ for ?

File: chsdi/models/__init__.py (hunk 1 above)
Comment (loicgasser) [inline chsdi/models/__init__.py]: yes, on first request match, here I reorder according to the scale (the ones that match the scale first) but I don't filter them anymore.
The break statement is in `_get_feature_db`

**Comment by loicgasser** [comment]
scan db is user:

1. To determine which bodId is associated to which geotable
2. To determine which layer has duplicated primary keys accross the tables

File: chsdi/models/__init__.py (hunk 1 above)
Comment (procrastinatio) [inline chsdi/models/__init__.py]: OK, I see

**Comment by loicgasser** [comment]
thanks @procrastinatio

**Pull request merged**